src/components/mgt-tasks/mgt-tasks.scss

```diff
@@ -123,99 +118,234 @@ mgt-tasks .Header .NewTaskButton,
+
+        .TaskDetail {
           width: 100%;
-          font-size: 20px;
-          font-weight: 300;
-        }
+          height: 100%;
+          margin: auto;
 
-        .TaskDetails {
           display: flex;
-          margin: 10px 0 0 0;
-          font-size: 12px;
-          font-weight: 400;
+          justify-content: flex-start;
           align-items: center;
-          flex-wrap: wrap;
+        }
 
-          svg {
-            vertical-align: middle;
-            margin-right: 4px;
+        .TaskTitle {
+          color: var(--task-title-color, $ms-color-gray160);
+          font-size: 14px;
+          font-weight: 600;
+          width: 240px;
+          padding: 12.5px 0 0 0;
+          grid-row: 1;
+          grid-column: 1 / 3;
+        }
+
+        .TaskGroup {
+          min-height: 24px;
+
+          grid-row: 2;
+          grid-column: 1;
+        }
+
+        .TaskBucket {
+          min-height: 24px;
+
+          grid-row: 2;
+          grid-column: 2;
+        }
+
+        .TaskAssignee {
+          grid-row: 3;
+          grid-column: 1;
+
+          mgt-people {
+            --list-margin: 0;
           }
-          span,
-          select {
-            vertical-align: middle;
+        }
+
+        .TaskDue {
+          justify-content: flex-end;
+          align-items: flex-start;
+
+          grid-row: 4;
+          grid-column: 1 / 3;
+          margin-bottom: 10px;
+        }
+
+        @media (min-width: 768px) {
+          grid-template-columns: 1fr 0.5fr 1fr 0.5fr;
+          grid-template-rows: auto auto;
+
+          .TaskTitle {
+            grid-row: 1;
+            grid-column: 1 / 5;
           }
-          .TaskDetail {
-            margin: 0 20px 0 0;
+
+          .TaskGroup {
+            grid-row: 2;
+            grid-column: 1;
           }
 
-          .TaskIcon {
-            color: #333333;
-            margin: var(--task-detail-icon-margin 0 8px 0 0);
+          .TaskBucket {
+            grid-row: 2;
+            grid-column: 2;
           }
-          mgt-people {
-            color: var(--task-detail-color, black);
-            font-size: 16px;
+
+          .TaskAssignee {
+            grid-row: 2;
+            grid-column: 3;
           }
 
-          mgt-person {
-            display: inline-block;
+          .TaskDue {
+            justify-content: flex-start;
+            align-items: center;
+
+            grid-row: 2;
+            grid-column: 4;
           }
-          .Picker {
-            background-color: white;
-            box-shadow: 0 3px 7px rgba(0, 0, 0, 0.3);
-            -webkit-background-clip: padding-box;
-            -moz-background-clip: padding-box;
-            background-clip: padding-box;
-            width: 350px;
-            color: var(--task-detail-color, black);
-            margin: 8px;
-            mgt-people-picker {
-              --separator-margin: 0px 10px 0px 10px;
-            }
+        }
+
+        @media (min-width: 1200px) {
```

Comment:
this will only work when the component is taking up the entire width of the page. We need a solution that works based on the size of the component, and not the size of the page. Take a look at how agenda is doing this.